helloworld-html5/README.md

```diff
@@ -63,8 +63,8 @@ The application will be running at the following URL <http://localhost:8080/jbos
 
 You can also test the REST endpoint as follows. Feel free to replace `YOUR_NAME` with a name of your choosing.
 
-* The *XML* content can be tested by accessing the following URL: <http://localhost:8080/jboss-helloworld-html5/hello/xml/YOUR_NAME> 
-* The *JSON* content can be tested by accessing this URL: <http://localhost:8080/jboss-helloworld-html5/hello/json/YOUR_NAME>
+* The *XML* content can be tested by accessing the following URL using HTTP POST: <http://localhost:8080/jboss-helloworld-html5/hello/xml/YOUR_NAME> 
+* The *JSON* content can be tested by accessing this URL using HTTP POST: <http://localhost:8080/jboss-helloworld-html5/hello/json/YOUR_NAME>
```

Comment:
@trepel : All of the quickstarts assume the they should use a browser to access the application URLs. While it uses HTTP POST, I'm afraid this language might confuse some users. They may think they need to use a CURL command. 

I wonder if we should make it clear that we intend them to access the URLs in a browser. WDYT?